src/util/ImageUploader.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
It would be nice with a tiny comment for each of the 4 functions you've made. It does not have to be a long comment, but just a tiny insight into what it does (e.g. "Exports settings from Chatterino to the ShareX export JSON format" or something)